modules/quadbin/snowflake/sql/QUADBIN_BBOX.sql

```diff
@@ -0,0 +1,148 @@
+-- This method should call QUADBIN_TOZXY, but that causes SnowFlake to crash,
+-- so we are computing the ZXY here, in-place
+CREATE OR REPLACE FUNCTION QUADBIN_BBOX
+(quadbin INT)
+RETURNS ARRAY
+AS $$
+WITH __interleaved AS (
+    SELECT
+        BITAND(BITSHIFTRIGHT(quadbin, 59), 7) AS mode,
+        BITAND(BITSHIFTRIGHT(quadbin, 57), 3) AS extra,
+        BITAND(BITSHIFTRIGHT(quadbin, 52), 31) AS z,
+        BITSHIFTLEFT(
+            BITAND(
+                quadbin,
+                TO_NUMBER('fffffffffffffff', 'XXXXXXXXXXXXXXXX')
```

Comment:
I guess this is very minor, but I would use the corresponding decimal constant (like 1152921504606846975 in this case) to remove a few TO_NUMBER calls (maybe SF is smart enough to optimize that away, but just in case...)